### ISSUE TITLE
async actions triggered with wp-cron don't call function

### ISSUE THREAD

**Issue by Nemi5150** [woocommerce/action-scheduler#691]
I have some code that queue's an async action to send off an email. It calls a function called `send_notice()`. When I schedule an action by calling `as_enqueue_async_action` and it triggers, if I see a log entry at `WooCommerce->Status->Scheduled Actions` with `action complete via Async Request` then all is good and the function is called. 

However, if the action gets called by `action complete via WP Cron` then it shows up in the action scheduler log, but the send_notice function is never called. I can put a log statements as the first line of the function but it never executes. Am I missing something silly?

tldr; when an action is called via an admin page load, it fires, but when it loads via a cron job (page without admin logged in) then it does not call the intended function.

I am using the Action Scheduler that is built into WooCommerce on my install.

**Comment by Nemi5150** [comment]
Not sure what is going on. I am getting some weird behavior from this, but I restarted my server and things appear to be working as of right now. I will reopen if I still have issues.
